### ISSUE TITLE
Value Flows logo

### ISSUE THREAD

**Issue by ahdinosaur** [valueflows/valueflows#38]
obligatory logo thread, where we throw out random ideas for how to represent Value Flows as a scalable vector graphic (SVG).

personally, my first sense of value flows are the images presented in [valnet/wiki/Value-Streams](https://github.com/valnet/valuenetwork/wiki/Value-Streams) and [valnet/wiki/Everything-is-connected](https://github.com/valnet/valuenetwork/wiki/Everything-is-connected), specifically:

![Sensorica value streams](https://camo.githubusercontent.com/aa0ca205c1f7280fc82f869899b0ea05e5bde4fb/68747470733a2f2f7261772e6769746875622e636f6d2f6268617567656e2f76616c75656e6574776f726b2f6d61737465722f76616c75656e6574776f726b2f736974655f6d656469612f6d656469612f70686f746f732f76616c75655f6e6574776f726b2e706e67)

and

[![Hypergraphic user interface](https://cloud.githubusercontent.com/assets/719605/10327163/b9f1c74c-6d00-11e5-943c-179e4025b1e9.png)](https://docs.google.com/drawings/d/14Ochj955jSTm2Upoa5VK6YfWkSlFGVWIEBdJ1WSJuxE/edit)

when i have time and feel a strong urge i'll try and draft some logos based on the above with a bit of Inkscape magic.

keen to hear what others have in mind.

**Comment by ahdinosaur** [comment]
it's also possible we post this on the [Open Source Design](http://opensourcedesign.net/) jobs board.

**Comment by bhaugen** [comment]
While I like both of those images, they are too big/complex for a logo. For inspiration, google for images for economic network diagram. What's the smallest network diagram that gets the idea across?

And what will holodex use?

**Comment by ahdinosaur** [comment]
reframed this issue to be about communicating Value Flows through visual design instead of specifically focusing on the logo format.

**Comment by elioqoshi** [comment]
Here are 2 proposals by me. What do you think?
![1](https://cloud.githubusercontent.com/assets/5436686/10416473/98f7d7c8-7018-11e5-9400-436a7eea6f32.jpg)

**Comment by bhaugen** [comment]
@elioqoshi - thank you very much for a great contribution! I love the look of the V. Suggests a mini-network.

I hate to second-guess people who design so much better than me (as in, I have no design skills at all), but could I suggest a tweak? Could you make it more flow-y somehow? What do you think? Not so much the V, but maybe the Valueflows font?

What do other people think? I just asked @fosterlynn. She likes the font, and she actually has some design skills. Maybe I should just go with the flow...

**Comment by elioqoshi** [comment]
Hey @bhaugen thanks for the kind words!
I used an open font here (Exo) to stay true with the open source design philosophy. I will see if I can come up with a rounder font here, let me check :)

**Comment by bhaugen** [comment]
@elioqoshi - why don't you wait for some other people to chime in before doing any extra work? I am famous for having no taste...

**Comment by elioqoshi** [comment]
haha no worries @bhaugen I'm happy to help.
I used another rounder typeface here (Fira Sans, used by Mozilla, also open source).
What do you prefer?
![2](https://cloud.githubusercontent.com/assets/5436686/10416724/1aec0354-7022-11e5-98e4-6368f77d0dc4.jpg)

**Comment by bhaugen** [comment]
I'm waiting for other opinions. See my previous comment above :wink:

**Comment by elioqoshi** [comment]
Roger then.

**Comment by ahdinosaur** [comment]
wow @elioqoshi this is awesome, i really like the V icon, thanks again for the sweet contribution. :cat:

as for the fonts, appreciate the use of open source fonts, i prefer the first one, but i understand @bhaugen's comment, i see it giving a more industrial vibe than flow-y. but i have equally bad taste in these details, more than happy to follow others' lead on this and not be a gatekeeper in the way of these being published on our website.

so @elioqoshi to keep this ball rolling, if you send me an `.svg` of the V icon, i'll pull request it into our [README](https://github.com/valueflows/valueflows/blob/master/README.md), which will automatically get published to [our website](https://valueflo.ws). or if you're feeling adventurous you can do this pull request yourself, i reckon add the `.svg` file to a new `assets` directory and link to it with [rawgit.com](https://rawgit.com/).

**Comment by elioqoshi** [comment]
@ahdinosaur Thanks for the kind words! I myself prefer the first one (it's the same font Atlassian uses). I also tried rounder fonts, but they're too cliche sci-fi (http://www.dafont.com/theme.php?cat=303&page=3)
If you'd ask me, I'd go ahead with the first proposal.

I'm still not a GitHub power user so I will attach the svg here :)

**Comment by fosterlynn** [comment]
@elioqoshi I "third" the excitement of having this awesome logo just appear in our issue!  Wow!  

I love the V (as Bob said).  I like the second V best (the white on blue one).  So I'd like to use that one primarily.  I'd like the keep the other one handy for different situations.  Does that make sense?

I like the first font better on the valueflows name, just because it is more interesting.  But I don't have strong opinions on that.

**Comment by elioqoshi** [comment]
Here we go:
https://dl.dropboxusercontent.com/u/62482708/Valueflows%20Icon.svg

P.S: Your website looks more like a text document as of now. Do you plan to power it with WordPress?

**Comment by elioqoshi** [comment]
@fosterlynn Exactly my thoughts. The V in a circle should be used primarily :)
Thank you!

**Comment by ahdinosaur** [comment]
> Here we go: https://dl.dropboxusercontent.com/u/62482708/Valueflows%20Icon.svg

cheers, one moment please.

> P.S: Your website looks more like a text document as of now.

hehe, baby steps. :footprints: it's a static site generating from our project content, only just got it working a few days ago, will be a whole lot better in time.

**Comment by elioqoshi** [comment]
@ahdinosaur Great! Looking forward to see the site updated then. Feel free to mention me in issues if you need design help

**Comment by fosterlynn** [comment]
@elioqoshi @ahdinosaur hmmmm thinking about the website question, which is a good one.  So what we have now is meant to be a human readable definition of the vocab, not so much a "website".  Are these the same or different?  I guess for schema.org (http://schema.org/) they are the same.  But for vocabs defined in the w3c arena, like http://www.w3.org/TR/activitystreams-vocabulary/, they just use the w3c format for the human readable vocab; and maybe have a separate website?  Or not?  Here is foaf's: http://www.foaf-project.org/, pretty minimal, and links to their vocab.

I guess we should have only one "website".  And maybe just links to the currently generated view(s).  Does this make sense?  I think I like the schema.org format best, a brief explanation of what we are trying to do on a home page, and sections or just links for vocab, etc.

Elio, do you want to help with the website design? :smile:  Please feel free to say No, sorry if I am pushy!  I can write some brief content for a front page for people's review.  And this is really no rush, we have lots of work to do before this is a vocab. I think for now, it would just be a place people could see where we are at if they are not github people.  But maybe they would want to connect.

**Comment by elioqoshi** [comment]
@fosterlynn I'd definitely like to! Just need to see if I have the time to do it. If it's WordPress, I can contribute a few hours here and there. What website structure are we looking for here?

**Comment by elioqoshi** [comment]
The theme I use on my website is pretty simple and flexible (and also free)
http://elioqoshi.me/

**Comment by ahdinosaur** [comment]
>  I can write some brief content for a front page for people's review. And this is really no rush, we have lots of work to do before this is a vocab. I think for now, it would just be a place people could see where we are at if they are not github people. But maybe they would want to connect. 

yes please! this is what i'd imagine would go in our [top-level README](https://github.com/valueflows/valueflows/blob/master/README.md), which is what is automatically published to [our website](https://github.com/valueflows/valueflo.ws).

@elioqoshi another option i think would help us here, depending on your interest, is to create more beautiful visualizations that might help communicate the concepts we're trying to describe in our technical vocabulary, e.g. through diagrams, artwork, sketches, etc. @bhaugen and @fosterlynn, if y'all have capacity i'd appreciate a showcase of your existing visuals in this thread. :)

**Comment by bhaugen** [comment]
Before showcasing too much, let's settle the rename of Economic Event, and I'll incorporate that into the existing visuals where required. In the meantime, @elioqoshi could look at the ones above in this thread.

**Comment by elioqoshi** [comment]
Do you want me to redesign existing diagrams? I lost a bit track here

**Comment by bhaugen** [comment]
@elioqoshi 

> Do you want me to redesign existing diagrams? I lost a bit track here

I don't think so, yet. We are doing some renaming, so if you worked on the existing diagrams, you might need to do some needless rework. I'll ping you when I think the existing diagrams are stable again, and then you can decide if you want to do some redesign.

But I do have a curiosity question: how did you find this project? I mean, how did we get so lucky to have such an awesome logo? (And thanks again.)

**Comment by elioqoshi** [comment]
@bhaugen I look from time to time at the Jobs section at Open Source Design :)
http://opensourcedesign.net/jobs/jobs/2015-10-08-valueflows.html

**Comment by bhaugen** [comment]
@ahdinosaur was so smart! Thanks, Mikey! Good move!

**Comment by fosterlynn** [comment]
@elioqoshi The website design maybe won't move too fast, but I do have a question: How would you like to work on such a thing?  For example, do you like do mockup designs, or do you like to start with say Wordpress and just do it, or do you like to dive into css and html?  Or I might be asking the question wrong, so feel free to reframe the question too!  I only know designers have different methods of work on this type of thing.

**Comment by fosterlynn** [comment]
> Me: I can write some brief content for a front page for people's review. And this is really no rush, we have lots of work to do before this is a vocab. I think for now, it would just be a place people could see where we are at if they are not github people. But maybe they would want to connect.
> 
> @ahdinosaur yes please! this is what i'd imagine would go in our top-level README, which is what is automatically published to our website.

So I looked at the readme, and it is pretty darn good right now.  I wanted to make sure that we clearly stated the current status so people don't think we have more than we have.  And I think the Roadmap section does that well enough.

**Comment by elioqoshi** [comment]
@fosterlynn I'm not a coder (although I'm experienced in WordPress) so I usually dive into Photoshop and start creating PSD's and mockups and later someone turns them into code :)

**Comment by fosterlynn** [comment]
@elioqoshi thanks!  Just seeing who would do what...  :)

**Comment by ahdinosaur** [comment]
> @fosterlynn I'm not a coder (although I'm experienced in WordPress) so I usually dive into Photoshop and start creating PSD's and mockups and later someone turns them into code :)

i'm happy to translate any mockups into code. :)

**Comment by ahdinosaur** [comment]
related to this issue, @bhaugen's [visual language experiments](https://docs.google.com/presentation/d/1N81Q0uDjA7OMEZTLNig6O-MLuWHgb5TL0IxPZ2U8qDU/edit#slide=id.gb7a109124_0_0).
